web-components/src/components/modal/Modal.ts

```diff
@@ -10,8 +10,8 @@ import "@/components/button/Button";
 import "@/components/icon/Icon";
 import { Key } from "@/constants";
 import { FocusTrapMixin } from "@/mixins";
-import reset from "@/wc_scss/reset.scss";
```

Comment:
Why is this removed? Be aware, the reset.scss contains styles that otherwise get messed up because of shadow-dom barriers, like `display`

Reply:
@futurethang  I think my VSCode must have auto-removed. Just few days back installed a plugin which actually auto updated the import and removes the one which are not being used

Reply:
I will insert it again if you want